.github/workflows/release.yml

```diff
@@ -325,4 +325,31 @@ jobs:
       - name: Remove release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: hub release delete ${{ needs.create_release.outputs.version }}
+        run: hub release delete ${{ needs.create_release.outputs.tag }}
+
+  send_slack_notification:
+    name: Notify the team Slack channel about this release
+    needs: [create_release, publish_release]
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    steps:
+      - name: Status is OK
+        if: ${{ needs.publish_release.outputs.status == 'success'  }}
+        run: |
+          cat <<EOF >> $GITHUB_ENV
+          SLACK_COLOR=success
+          SLACK_MESSAGE=:rocket: Published release ${{ needs.create_release.outputs.release }}.
+          EOF
+      - name: Status is FAILED
+        if: ${{ needs.publish_release.outputs.status != 'success'  }}
+        run: |
+          cat <<EOF >> $GITHUB_ENV
+          SLACK_COLOR=failure
+          SLACK_MESSAGE=:x: Release ${{ needs.create_release.outputs.release }} *FAILED*.
+          EOF
+      - name: Send notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Hope this only requires read-only privileges on GitHub.

Reply:
This does not actually access GitHub APIs, it calls hooks.slack.com.